example/eblob_backend.c

```diff
@@ -86,15 +86,28 @@ static int eblob_read_params_compare(const void *p1, const void *p2)
 }
 
 /* Pre-callback that formats arguments and calls ictl->callback */
-static int blob_iterate_callback_common(struct eblob_disk_control *dc, void *data, void *priv, int no_meta) {
+static int blob_iterate_callback_common(struct eblob_disk_control *dc, int fd, uint64_t data_offset, void *priv, int no_meta) {
 	struct dnet_iterator_ctl *ictl = priv;
 	struct dnet_ext_list elist;
 	struct eblob_backend_config *c = ictl->iterate_private;
 	uint64_t size;
+	void *data, *saved_data;
 	int err;
 
 	assert(dc != NULL);
-	assert(data != NULL);
+
+	size = dc->disk_size;
+	saved_data = data = malloc(size);
```

Comment:
I found this in current code example/eblob_backend.c at blob_iterate_callback_common:

	if (dc->flags & BLOB_DISK_CTL_EXTHDR) {
		size -= sizeof(struct dnet_ext_list_hdr);
	}

It seems that size decremented, but data pointer is not adjusted. Is it ok or should I add
data += sizeof(struct dnet_ext_list_hdr);
here?